cudax/include/cuda/experimental/__device/device.cuh

```diff
@@ -35,7 +37,22 @@ class device
 {
   int __id = 0;
 
+  template <::cudaDeviceAttr Attr>
+  struct __attr
```

Comment:
We conventionally use `__ugly` for variables in libcu++.

Can we go with `_Ugly` for structs? That makes it much easier to discern what is what

Reply:
Wwe descided to not care about that but rather wether something is dependent and make sure that is clear